tests/test_cli.py

```diff
@@ -190,6 +191,73 @@ def test_duplicate_entities(self):
                     assert bad_arg not in help_return, f"Found {bad_arg} in `accelerate launch -h`"
 
 
+class ClusterConfigTester(unittest.TestCase):
```

Comment:
Just for my understanding, these are general tests that don't relate to numactl specifically, right?